molten-test-mockito-autoconfigure/pom.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please also add a readme for this module.

Reply:
Added in 131832b5.